### PR TITLE
Implemented List Comments

### LINKED ISSUE #234: List Post/Comment Parity
closes #216

### PR DESCRIPTION
**Pull request by Pdzly**
closes #234

### CODE REVIEW

File: src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtFilter.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ protected void doFilterInternal(
       final HttpServletResponse response,
       final FilterChain filterChain) throws ServletException, IOException {
 
-    String authorizingToken = request.getHeader("Authorization");
+    String authorizingToken = request.getHeader("authorization");
```
Comment (jgrim) [inline src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtFilter.java:37]: Why did you make this lowercase? I think uppercase is the standard, no?

File: src/main/java/com/sublinks/sublinksapi/post/services/PostLikeService.java [hunk 2]
```diff
@@ -55,13 +57,8 @@ private void updateOrCreatePostLike(final Post post, final Person person, final
...
 
-    final PostLike postLike = PostLike.builder()
-        .post(post)
-        .person(person)
-        .isUpVote(score == 1)
-        .isDownVote(score == -1)
-        .score(score)
-        .build();
+    final PostLike postLike = PostLike.builder().post(post).person(person).isUpVote(score == 1)
+        .isDownVote(score == -1).score(score).build();
```
Comment (jgrim) [inline src/main/java/com/sublinks/sublinksapi/post/services/PostLikeService.java:61]: Why does your IDE collapse these down? I think it's more difficult to read this way.

File: src/main/java/com/sublinks/sublinksapi/comment/repositories/CommentLikeRepositoryImpl.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import jakarta.persistence.criteria.Root;
+import org.springframework.beans.factory.annotation.Autowired;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.sublinks.sublinksapi.utils.PaginationUtils.applyPagination;
+
+public class CommentLikeRepositoryImpl implements CommentLikeRepositorySearch {
+
+  @Autowired
```
Comment (jgrim) [inline src/main/java/com/sublinks/sublinksapi/comment/repositories/CommentLikeRepositoryImpl.java:21]: The style guide says to use constructor injection over field injection.

https://sublinks.org/docs/style_guide.html
Spring Boot Specific Conventions
> 9. Dependency Injection
>   Prefer constructor injection over field injection.

File: src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/post/mappers/PostVoteViewMapper.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.lang.Nullable;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
+public interface PostVoteViewMapper extends
+    Converter<com.sublinks.sublinksapi.post.dto.PostLike, VoteView> {
+
+  PersonMapper personMapper = Mappers.getMapper(PersonMapper.class);
+
+  @Override
+  @Mapping(target = "creator", expression = "java(personMapper.convert(postLike.getPerson()))")
```
Comment (jgrim) [inline src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/post/mappers/PostVoteViewMapper.java:21]: I don't believe you don't need to use this expression. It seems like you're attempting to use another mapper.

See: https://mapstruct.org/documentation/stable/reference/html/#invoking-other-mappers

**Review by jgrim — CHANGES_REQUESTED** [review]
Please consider changing to use the project's code styles for formatting. There are a few comments on things that need fixing.

File: src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtFilter.java (hunk 1 above)
Comment (Pdzly) [reply src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtFilter.java:37]: nope lemmy uses lowercase ;)

File: src/main/java/com/sublinks/sublinksapi/post/services/PostLikeService.java (hunk 2 above)
Comment (Pdzly) [reply src/main/java/com/sublinks/sublinksapi/post/services/PostLikeService.java:61]: Fixed it, there was something wrong in the editorconfig file

**Review by jgrim — APPROVED** [review]

**Pull request merged**